test.js

```diff
@@ -100,6 +100,28 @@ describe('test', function () {
     it('works with label statement', function () {
       assert.equal(require('./fixtures/label'), 1)
     })
+
+    it('ignores files excluded by `ignore` option', function () {
+      assert.ok(global.__coverage__[require.resolve('./fixtures/ignored')] === undefined)
```

Comment:
Wondering if it'd be better to test this one the same way the other two tests below are being tested and to extract this logic into a function to be shared. Thoughts?